src/amo/components/SearchResult.js

```diff
@@ -53,12 +53,16 @@ export class SearchResultBase extends React.Component {
               <Rating rating={addon.ratings.average} readOnly
                 styleName="small" />
             </div>
-            { addon.authors && addon.authors.length ?
-              <h3 className="SearchResult-author">{addon.authors[0].name}</h3> : null }
-            <h3 className="SearchResult-users">{i18n.sprintf(
-              i18n.ngettext('%(total)s user', '%(total)s users', averageDailyUsers),
-              { total: i18n.formatNumber(averageDailyUsers) },
-            )}
+            <h3 className="SearchResult-author-users">
+              {i18n.sprintf(i18n.ngettext(
+	  '%(addonAuthor)s %(separator)s %(addonUsers)s user', 
+	  '%(addonAuthor)s %(separator)s %(addonUsers)s users', 
+	  averageDailyUsers,
+	), {
+	  addonAuthor: addon.authors.length ? addon.authors[0].name : null,
```

Comment:
The test fail relates to this line because it assumes addon.authors is present. You can be a bit more defensive and do `addon.authors && addon.authors.length ? addon.authors[0].name : null` like the previous code. 

You should also keep the display of this part conditional otherwise if `addon.authors` is undefined you're still outputting the `h3.SearchResult-author-users` (once you fix the exception 😀).